atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -1849,13 +1849,14 @@ public void putUnlessExists(final TableReference tableRef, final Map<Cell, byte[
                             putUnlessExistsSinglePartition(tableRef, client, partition.getKey(), partition.getValue());
                     if (!casResult.isSuccess()) {
                         return Optional.of(new KeyAlreadyExistsException(
-                                String.format("The cells in table %s already exist.", tableRef.getQualifiedName()),
+                                "The cells in table {} already exist.",
```

Comment:
```suggestion
                                "The cells in the table already exist.",
```
Can drop interpolation since SafeExceptions.renderMessage appends info to the end. We only need to deal with interpolation for log lines